front/app/components/SignUpIn/SignUpInModal.tsx

```diff
@@ -84,7 +84,11 @@ const SignUpInModal = memo<Props>(({ className, onMounted }) => {
         setSignUpActiveStep(activeStep);
       }),
       changeMetaData$.subscribe(({ eventValue: metaData }) => {
-        setMetaData(metaData);
+        if (metaData.inModal) {
+          setMetaData(metaData);
+        } else {
+          onClose();
+        }
```

Comment:
Bit confused by this code. Are we sure we want co call `onClose` whenever the metaData changes and we're not in a modal? I can imagine this could potentially cause some bugs/unexpected behavior.

Reply:
Good point, will remove 👍